src/IB/IIMethod.cpp

```diff
@@ -414,6 +415,10 @@ IIMethod::preprocessIntegrateData(double current_time, double new_time, int /*nu
     d_U_new_vecs.resize(d_num_parts);
     d_U_half_vecs.resize(d_num_parts);
 
+    d_U_old_systems.resize(d_num_parts);
+    d_U_old_vecs.resize(d_num_parts);
+    d_U_old_updated_vecs.resize(d_num_parts);
```

Comment:
I think these should only be set up if the solver is configured to do multistep time stepping.

Reply:
Also, I think you should be able to avoid creating a new system. Please see the implementation in `IBFEMethod.cpp`, which uses an `"old"` context for the previous vectors. Creating a new system has a lot more overhead than just creating a new vector context.

Reply:
Ah, `IIMethod` has not yet been updated to use `LibMeshSystemVectors`. Nonetheless, you don't need a new `System` to hold the old value of the velocity.

Reply:
These vectors need to persist between timesteps, right? If so we need to put them in a system (or somewhere equivalent) so that they are available for restarts.

Reply:
Agh --- the `IBFEMethod` implementation probably is broken with restarts. 👎

Reply:
Or do named `System` vectors get written to restart? In `IBFEMethod`, the lagged value is stored as a new vector in the velocity system named `"old"`.

Reply:
Named systems are saved to libMesh's restart file so I think that's okay (but someone should write tests, naturally).

Reply:
Named vectors, right?

Reply:
I did some investigating and it looks like you have to set `EquationSystems::WriteFlags` to include `WRITE_ADDITIONAL_DATA` to get these vectors. I haven't tried it.

Reply:
We probably need to add a test for the multistep IBFE implementation.